moreutils/moreutils.py

```diff
@@ -83,18 +123,20 @@ async def _thetime(self, ctx):
     async def color(self, ctx, *, color: discord.Color):
         """Shows some info about provided color."""
         colorrgb = color.to_rgb()
-        colorhsv = colorsys.rgb_to_hsv(colorrgb[0], colorrgb[1], colorrgb[2])
-        colorhls = colorsys.rgb_to_hls(colorrgb[0], colorrgb[1], colorrgb[2])
-        coloryiq = colorsys.rgb_to_yiq(colorrgb[0], colorrgb[1], colorrgb[2])
+        r, g, b = [x / 255.0 for x in colorrgb]
```

Comment:
```suggestion
        r, g, b = [x / 255 for x in colorrgb]
```
i dont see any particular point in specifying float here

also probably better for readability make that a one variable and then just use `*var` in functions below. Smth like
```py
rgb_coords = [x / 255 for x in colorrgb]
coloryiq = colorsys.rgb_to_yiq(*rgb_coords)

Reply:
Ok I have made these changes as we discussed in DM. 👍🏼